frontend/src/components/UserProfile/UserProfileEditAvatar.tsx

```diff
@@ -26,7 +26,7 @@ export const _UserProfileEditAvatar = ({ userUpdateProfileAvatar }: Props) => {
   const [success, setSuccess] = useState(false);
   const [msg, setMsg] = useState('');
 
-  const [loading, setLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
You can set the default value to true so that you won't call the `setIsLoading` in useEffect

Reply:
sir i think it should be false since im not using it inside a useEffect
though i am using it in `handleFileChange` function which will need the isloading to be changed to true when handling the request

Reply:
Okay, I thought you are using that for loading state(pending)